libs/coin-framework/src/api/types.ts

```diff
@@ -42,6 +42,7 @@ export type Operation<
    * This can include things like status, error messages, swap info, etc.
    */
   details?: Record<string, unknown>;
+  assetInfo?: AssetInfo;
```

Comment:
We now have `asset` and `assetInfo` in the type 🤔

Reply:
No I think it's an oversight